### PR TITLE
[eclipse/xtext-eclipse#1022] sort quickfixes by label to have a stable order if multiple quickfixes are available

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext-eclipse#1022] sort quickfixes by label to have a stable order if multiple quickfixes are available
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend [hunk 1]
```diff
@@ -116,7 +116,7 @@ abstract class AbstractQuickfixTest extends AbstractEditorTest {
 		val originalText = document.get
 		val issue = document.getValidationIssue(issueCode)
 
-		val actualIssueResolutions = issue.getResolutions
+		val actualIssueResolutions = issue.getResolutions.sortBy[label]
```
Comment (miklossy) [inline org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend]: I would sort not only the actual, but also the expected quickfixes.

Comment (cdietrich) [reply org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend]: Well there is no additional value in that. Thus I keep it one side

Comment (cdietrich) [reply org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend]: You can come up with a second or after this one

Comment (miklossy) [reply org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend]: If you sort the expected values here, there is no need to modify the concrete test cases.

Comment (cdietrich) [reply org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend]: Can you please come up with ape and we will close this one

**Pull request closed without merging**